src/main/java/Nearby.java

```diff
@@ -0,0 +1,40 @@
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Created by marti on 2017.05.23..
+ */
+public class Nearby {
+
+    private int[][] multi = new int[][]{
+            { 2, 0, 4, 1241, 12, 5, 11, 1110, -42, 424, 1, 12323 },
+            { 1, 3, 5, 7 },
+            { 321, 320, 32, 3, 41241, -11, -12, -13, -66, -688 }
+    };
+
+    public int[] nearby(int x, int y, int range) {
+        List<Integer> resultList = new ArrayList<>();
+        if (x > multi.length ) throw new IllegalArgumentException("There is no row: " + x );
+        if (y > multi[x].length ) throw new IllegalArgumentException("There is no column: " + y);
+        for (int j = 0; j < multi[x].length; j++) {
+            if (j >= (y - range) && j <= (y + range) && j!=y) {
```

Comment:
miert nem veszed bele a feltetelt a for-ba? gyorsabb mert nem iteral foloslegesen. pl:
```
start = max(0, y-range)
stop = min(y+range, length-1)
for (i=start, i <=stop, i++) {
    if (i!=y) add
}
```